dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/internal/net/TracingRequestInterceptorTest.kt

```diff
@@ -129,10 +131,13 @@ internal class TracingRequestInterceptorTest {
     @RegexForgery("\\d(\\.\\d){3}")
     lateinit var fakePackageVersion: String
 
+    lateinit var tracedRequestListener: TracedRequestListener
```

Comment:
Why are you creating a spy and not simply a Mock of the interface ?

Reply:
I am not creating a spy...the spy is around the TracingRequestInterceptor and that was for another reason I guess.